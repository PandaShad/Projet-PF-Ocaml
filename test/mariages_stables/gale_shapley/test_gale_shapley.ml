open Mariages_stables
open Mariages_stables_exemple
open Exemple

let () = run Gale_shapley.algo
