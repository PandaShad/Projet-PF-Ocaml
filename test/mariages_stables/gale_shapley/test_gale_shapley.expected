EXEMPLE Knuth pages 12 et 22-26
ENTREE {
  n=4
  A : c,b,d,a
  B : b,a,c,d
  C : b,d,a,c
  D : c,a,d,b
  a : A,B,D,C
  b : C,A,D,B
  c : C,B,D,A
  d : B,A,C,D
}
CONFIGURATION {
  A1 B1 C0 D0
  a? bC cD d?
}
CONFIGURATION {
  A2 B1 C0 D0
  aB bC cD d?
}
CONFIGURATION {
  A2 B1 C0 D0
  aB bC cD dA
}
SORTIE { Ad Ba Cb Dc }
